src/pages/tutorial/react/step-1.mdx

```diff
@@ -157,38 +153,30 @@ In the `src` directory, rename `index.css` as `index.scss`. Then in `index.js` u
 
 In `index.scss`, import the Carbon styles by adding the following at the top of the file:
 
-##### src/index.scss
-
-```scss
+```scss path=src/index.scss
 @import 'carbon-components/scss/globals/scss/styles.scss';
 ```
 
 This will take a moment for the Sass to compile. Once compiled, you'll notice that the Carbon base styling is applied (IBM Plex Sans font family, font size, weight, colors, etc.)
 
 Because any change to `index.scss` will re-compile all of the Carbon Sass, create an `app.scss` file in the `src` directory and in `App.js`, import that new file.
 
-##### src/App.js
-
-```javascript
+```javascript path= path=src/App.js
```

Comment:
I see a couple of these. I'd find all `path= ` to remove.

Reply:
just pushed something that removes them, hopefully I got them all!